README.rst

```diff
@@ -97,7 +97,8 @@ This project depends on
 
 1. `Spyder <https://github.com/spyder-ide/spyder>`_
 2. `Tornado <https://github.com/tornadoweb/tornado>`_
-3. `Pexpect <https://github.com/pexpect/pexpect>`_
+3. `Pexpect <https://github.com/pexpect/pexpect>`_ (*nix Systems)
+4. `pywinpty <https://github.com/spyder-ide/pywinpty>` (Windows Systems)
```

Comment:
It seems my last comment was not registered correctly: please add a missing `_` at the end here, i.e.

    pywinpty <https://github.com/spyder-ide/pywinpty>`_ (Windows Systems)